plugin-java/src/main/java/appland/execution/AppMapJavaConfigUtil.java

```diff
@@ -50,24 +33,18 @@ private AppMapJavaConfigUtil() {
                                                                           @NotNull VirtualFile context) {
         ApplicationManager.getApplication().assertReadAccessAllowed();
 
-        // content root of the module, which contains the context file
-        var matchingModuleRoot = Arrays.stream(ModuleRootManager.getInstance(module).getContentRoots())
-                .filter(root -> VfsUtilCore.isAncestor(root, context, false))
-                .findFirst()
-                .orElse(null);
-        if (matchingModuleRoot != null) {
-            return matchingModuleRoot;
-        }
-
-        // Some project models, e.g. Gradle, create multiple modules for a single gradle project.
-        // For example, Gradle module "main" with content root "<root>/module-one/src/main"
-        // and a top-level module with content root "<root>/module-one".
-        // The context (e.g. "<root>/module-one") may be located outside the root of module (e.g "main").
-        var projectContentRoot = ProjectRootManager.getInstance(module.getProject())
-                .getFileIndex()
-                .getContentRootForFile(context, false);
-        if (projectContentRoot != null) {
-            return projectContentRoot;
+        // Any of the module's roots will do
+        var moduleRoot = ModuleRootManager.getInstance(module).getContentRoots()[0];
```

Comment:
I don't think there's a guarantee that a module contains at least one content root.